TTS/tts/models/vits.py

```diff
@@ -41,6 +41,23 @@
 mel_basis = {}
 
 
+@torch.no_grad()
+def weights_reset(m: nn.Module):
+    # check if the current module has reset_parameters and if it is reset the weight
+    reset_parameters = getattr(m, "reset_parameters", None)
+    if callable(reset_parameters):
+        m.reset_parameters()
+
+
+def get_module_weights_sum(mdl: nn.Module):
+    dict_sums = {}
+    for name, w in mdl.named_parameters():
+        if "weight" in name:
+            value = w.sum().item()
```

Comment:
you might need to w.data to prevent any overlooked autograd hook

Reply:
Indeed good catch. Done